scrapfly/client.py

```diff
@@ -748,6 +781,8 @@ def _handle_api_response(
             else:
                 body = response.content.decode('utf-8')
 
+        body = self._handle_scrape_large_objects(body=body)
```

Comment:
Can we apply the logic only if that's a large object rather than calling it all time and doing no-op inside the function if that not the case to keep a consistent logic

```
if content_format in ['clob', 'blob']:
    body = self._handle_scrape_large_objects(body=body, format=content_format)
```

Reply:
yes it's more clear now 👍